contracts/src/v0.7/UpkeepRegistrationRequests.sol

```diff
@@ -0,0 +1,411 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.7.6;
+
+import "./interfaces/LinkTokenInterface.sol";
+import "./vendor/Owned.sol";
+import "./KeeperRegistryInterface.sol";
+import "./SafeMath96.sol";
+
+/**
+ * @notice Contract to accept requests for upkeep registrations
+ * @dev There are 2 registration workflows in this contract
+ * Flow 1. auto approve OFF / manual registration - UI calls `register` function on this contract, this contract owner at a later time then manually
+ *  calls `approve` to register upkeep and emit events to inform UI and others interested.
+ * Flow 2. auto approve ON / real time registration - UI calls `register` function as before, which calls the `registerUpkeep` function directly on
+ *  keeper registry and then emits approved event to finish the flow automatically without manual intervention.
+ * The idea is to have same interface(functions,events) for UI or anyone using this contract irrespective of auto approve being enabled or not.
+ * they can just listen to `RegistrationRequested` & `RegistrationApproved` events and know the status on registrations.
+ */
+contract UpkeepRegistrationRequests is Owned {
+    using SafeMath96 for uint96;
```

Comment:
I notice that some files are tab-delimited, and some are two space-delimited. Can we formalise to two spaces across all files, please?

Reply:
done